services/github/src/constants.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  CONCLUSIONS: { // maps status communicated by the queue service to github checkrun conclusions
+    /*eslint quote-props: ["error", "consistent-as-needed"]*/
+    'completed': 'success',
+    'failed': 'failure',
+    'exception': 'failure',
+    'deadline-exceeded': 'timed_out',
+    'canceled': 'cancelled',
+    'superseded': 'neutral', // queue status means: is not relevant anymore
+    'claim-expired': 'failure',
+    'worker-shutdown': 'neutral', // queue status means: will be retried
+    'malformed-payload': 'action_required', // github status means "correct your task definition"
+    'resource-unavailable': 'failure',
+    'internal-error': 'failure',
+    'intermittent-task': 'neutral', // queue status means: will be retried
+  },
+  CHECKRUN_TEXT: 'View task in Taskcluster',
+  CUSTOM_CHECKRUN_TEXT_ARTIFACT_NAME: 'public/customCheckRunText.md',
```

Comment:
Tom had suggested `public/github/customCheckRunText.md` here, and I think that makes sense as a parallel to `public/logs/`, our conventional place for task logfiles.

Reply:
good point, I missed that from his comment, thanks